core.yaml

```diff
@@ -1,3 +1,4 @@
+# NOTE: This file is merged with other values in ./helmfile.d/snippets/env.gotmpl
 toolsVersion: 1.4.9
```

Comment:
this line has to go

Reply:
That line was suggested by @j-zimnowoda

Reply:
yes, I suggested it because it is not obvious that `core.yaml` is part of merged values.
Our colleagues spend a significant time on finding out the root cause of the validation issue, because they did not know about that. I think it is worth to have this command if it can save someone's time in the future.